obspy/io/rg16/__init__.py

```diff
@@ -36,32 +36,32 @@
 .. figure:: /_images/rg16_node_orientation.png
 
 Reading the waveforms
--------
+---------------------
 The rg16 format can be read using two methods:
 
-1. Using the standard :func:`~obspy.core.stream.read` function. Optionally,
+1. Using the standard :func:``~obspy.core.stream.read`` function. Optionally,
    the format parameter can be specified as "rg16" for a speed-up.
 
-2. Using the :mod:`obspy.io.rg16` specific function
-   :func:`obspy.io.rg16.core._read_rg16`.
+2. Using the :mod:``obspy.io.rg16`` specific function
+   :func:``obspy.io.rg16.core._read_rg16``.
```

Comment:
same here @thefroid these are internal sphinx links, see http://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#python-roles  (although we just use `:func:` and not `:py:func:`), so they need exactly one backtick.